python/tskit/trees.py

```diff
@@ -1194,10 +1194,191 @@ def draw_text(self, orientation=None, **kwargs):
+                .tree .edges .p10 {stroke: red}
+
+            Mutations can be targetted by id, site id, or node number. The following
+            style displays all mutations immediately above node 10 as yellow with a black
+            border, and decreases the height of all mutation symbols on the tree,
+            displaying them as only 2 pixels high, rather than the default of 6.
+
+            .. code-block:: css
+
+                .tree .symbols .mutations .n10 {fill: yellow; stroke: black}
+                .tree .symbols .mutations rect {width: 6px; height: 2px;
+                    /* also must re-centre the rect */ transform: translate(-3px, -1px);}
+
+        :param str path: The path to the file to write the output. If None, do not
+            write to file.
+        :param size: A tuple of (width, height) giving the width and height of the
+            produced SVG drawing in abstract user units (usually interpreted as pixels on
+            initial display). If None, a default of (200, 200) is used.
+        :type size: tuple(int, int)
+        :param str tree_height_scale: Control how height values for nodes are computed.
+            If this is equal to ``"time"`` (the default), node heights are proportional
+            to their time values. If this is equal to ``"log_time"``, node heights are
+            proportional to their log(time) values. If it is equal to ``"rank"``, node
+            heights are spaced equally according to their ranked times.
+        :param str,float max_tree_height: The maximum tree height value in the current
+            scaling system (see ``tree_height_scale``). Can be either a string or a
+            numeric value. If equal to ``"tree"`` (the default), the maximum tree height
+            is set to be that of the oldest root in the tree. If equal to ``"ts"`` the
+            maximum height is set to be the height of the oldest root in the tree
+            sequence; this is useful when drawing trees from the same tree sequence as it
+            ensures that node heights are consistent. If a numeric value, this is used as
+            the maximum tree height by which to scale other nodes.
+        :param node_labels: If specified, show custom labels for the nodes
+            (specified by ID) that are present in this map; any nodes not present will
+            not have a label.
+        :type node_labels: dict(int, str)
+        :param mutation_labels: If specified, show custom labels for the
+            mutations (specified by ID) that are present in the map; any mutations
+            not present will not have a label.
+        :type mutation_labels: dict(int, str)
+        :param node_attrs: Set custom attributes for specific node symbols. Each key in
+            the `node_attrs` map is a node id; the corresponding value must be a
+            dictionary specifying additional parameters passed to the
+            :meth:`svgwrite.drawing.Drawing.circle` method.
```

Comment:
I'm not sure we want to tie ourselves either to a specific shape for nodes/mutation or to the ``svgwrite`` package long term. I'd much prefer to keep it generic and defined in terms of SVG.

The simpler thing to do there is to just drop these ``x_attrs`` parameters from the API for now.

Reply:
ISWYM. Shall I simply undocument them, but leave them hidden in there?

Reply:
Yeah, just undocument. That way we can use them and develop them a bit before commiting to a long-term API

Reply:
Done